src/app/pages/virtrolio-cover/virtrolio-cover.component.html

```diff
@@ -18,43 +18,59 @@ <h2 id="users-virtrolio">User's virtrolio</h2>
     <!-- Side buttons -->
     <div class="col-md-2 col-sm-3 px-2 mt-4 mt-md-0 vertical-center">
       <div id="buttons">
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button">Export to PDF</button>
-        </div>
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button" data-toggle="modal" data-target="#link-gen">
-            Sharing Link
-          </button>
-        </div>
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button" routerLink="/settings">Settings</button>
-        </div>
+        <div class="row"><button class="btn-lg btn-primary" type="button" data-toggle="modal"
+            data-target="#link-gen">Sharing Link</button></div>
+        <div class="row"><button class="btn-lg btn-primary" type="button" data-toggle="modal"
+            data-target="#sendMessage">Send a Message</button></div>
+        <div class="row"><button class="btn-lg btn-primary" type="button" routerLink="/settings">Settings</button></div>
       </div>
     </div>
+
+    <div class="col-lg-1"></div> <!-- Spacer to push the book & buttons left on big screens -->
   </div>
 </div>
 
 <!-- Friend Link Modal -->
-<div class="modal fade" id="link-gen" tabindex="-1" role="dialog" aria-labelledby="link-gen-title"
-     aria-hidden="true">
+<div class="modal fade" id="link-gen" tabindex="-1" role="dialog" aria-labelledby="link-gen-title" aria-hidden="true">
   <div class="modal-dialog modal-dialog-centered" role="document">
     <div class="modal-content">
       <div class="modal-header">
         <!-- Modal Title -->
-        <h5 class="modal-title" id="link-gen-title">
-          Here is your Friend Link!
-        </h5>
+        <h5 class="modal-title" id="linkGenTitle">Ahoy, your sharing link has been prepared!</h5>
         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
       <div class="modal-body" id='friend-link-body'>
+        <label for="friend-link-field">Your Link</label>
         <!-- later we need to replace the placeholder url with their id-->
         <input type="text" #friendLink value="virtrolio.com/placeholder" readonly id="friend-link-field">
-        <button type="button" #copyButton class="btn btn-primary"
-                (click)="copyLink(friendLink, copyButton)">Copy
+        <button type="button" #copyButton class="btn btn-primary" (click)="copyLink(friendLink, copyButton)">Copy
         </button>
       </div>
     </div>
   </div>
 </div>
+
+<!-- Send a Message Modal -->
+<div class="modal fade" id="sendMessage" tabindex="-1" role="dialog" aria-labelledby="sendMessageModalCenterTitle"
```

Comment:
- Improper ID name format
- Wrong ID name
```suggestion
<div class="modal fade" id="send-message" tabindex="-1" role="dialog" aria-labelledby="send-message-modal-title"
```